packages/babel-traverse/src/path/lib/virtual-types-validator.ts

```diff
@@ -201,6 +210,7 @@ export function isForAwaitStatement(this: NodePath): boolean {
   return isForOfStatement(this.node, { await: true });
 }
 
+// TODO: Remove in Babel 8
 export function isExistentialTypeParam(this: NodePath): void {
   throw new Error(
     "`path.isExistentialTypeParam` has been renamed to `path.isExistsTypeAnnotation()` in Babel 7.",
```

Comment:
Let's remove it now?

Reply:
Yeah we can use something like https://github.com/babel/babel/blob/9ca034f678d74afa07142baba24564a328cdb5fb/packages/babel-core/src/index.ts#L84-L88